runtime/vm/extendedMessageNPE.cpp

```diff
@@ -1605,21 +1605,29 @@ simulateStack(J9NPEMessageData *npeMsgData)
 		case RTV_SEND: {
 			J9UTF8 *classSig = NULL;
 
+			Trc_VM_simulateStack_RTVSEND_bcIndex(vmThread, currentBytecode, bcIndex);
 			if (JBinvokeinterface2 == currentBytecode) {
-				/* Set to point to JBinvokeinterface */
+				/* Set to point to JBinvokeinterface. */
 				bcIndex += 2;
+				Trc_VM_simulateStack_RTVSEND_bcIndex2(vmThread, bcIndex);
 			}
 			UDATA index = PARAM_16(bcIndex, 1);
+			Trc_VM_simulateStack_RTVSEND_index(vmThread, index);
 			if (JBinvokestaticsplit == currentBytecode) {
 				index = *(U_16 *)(J9ROMCLASS_STATICSPLITMETHODREFINDEXES(romClass) + index);
+				Trc_VM_simulateStack_RTVSEND_JBinvokestaticsplit_index(vmThread, index);
 			} else if (JBinvokespecialsplit == currentBytecode) {
 				index = *(U_16 *)(J9ROMCLASS_SPECIALSPLITMETHODREFINDEXES(romClass) + index);
-			} else if (JBinvokedynamic == currentBytecode) {
+				Trc_VM_simulateStack_RTVSEND_JBinvokespecialsplit_index(vmThread, index);
+			}
+			if (JBinvokedynamic == currentBytecode) {
```

Comment:
Ah, is it so that the else code below executes for all the other bytecodes?

Reply:
This was the problem, the `index` calculated in `JBinvokestaticsplit`/`JBinvokespecialsplit` block should be used by later `JBinvokedynamic` and others, otherwise `classSig` won't be extracted and stays the initial value `NULL`.

Reply:
> Ah, is it so that the else code below executes for all the other bytecodes?

Yes